app/forms/apo_form.rb

```diff
@@ -79,6 +79,19 @@ def collection_radio
     'none'
   end
 
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Open to suggestions on how to refactor this method to remove these rubocop exceptions.

Reply:
how about doing something like `default_access = model.administrative.defaultAccess` and using that local var throughout. That will likely reduce complexity substantially.

Reply:
Thanks! That changed helped drop the complexity to 8/7. Moved some of the code to the private `check_default_object_rights` method and now passes.